### DISCUSSION TITLE
Usability of date picker

### DISCUSSION THREAD

**Discussion by islandmix** [HiEventsDev/Hi.Events#181] [General]
I noticed this while testing with desktop... but let me know if this is working as expected

1. Usability: After selecting the date and time, it is not intuitive that you need to click somewhere else to get rid of the date picker tool window. I have seen other platforms where Month, Date and Time are separate and after you click time then it takes you to next field (end date) ... It would get a bit annoying for a new user after they set the end date/time and the popup is still there... this happens for all start date and end date fields... eg) create ticket, create event
![image](https://github.com/user-attachments/assets/1a5f5bda-1ece-471b-a93f-b238786f467d)

Or.. can't we just keep it simple like event brite? They even incorporate multiple days within 3 visible fields
![image](https://github.com/user-attachments/assets/150d1040-5f9f-423e-a82c-83c3b8f5a958)

**Comment by daveearley** [comment]
The [native `datetime-local`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local) input is being used for the date fields, so the behavior you're noticing is expected.

I chose the native date picker because it doesn’t require an additional library and works reliably. While it might require an extra click to close the picker, I prefer this over the complexity of using a third-party library.

If this becomes a concern for more users, I'll definitely consider alternatives.
